rectools/models/rank/rank_torch.py

```diff
@@ -0,0 +1,219 @@
+#  Copyright 2024-2025 MTS (Mobile Telesystems)
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Torch ranker model."""
+
+import typing as tp
+
+import numpy as np
+import torch
+from scipy import sparse
+from torch.utils.data import DataLoader, TensorDataset
+
+from rectools import InternalIds
+from rectools.models.base import Scores
+from rectools.models.rank.rank import Distance
+from rectools.types import InternalIdsArray
+
+
+class TorchRanker:
+    """
+    Ranker model based on torch.
+
+    This ranker is suitable for the following cases of scores calculation:
+    1. subject_embeddings.dot(objects_embeddings)
+    2. subject_interactions.dot(item-item-similarities)
+
+    Parameters
+    ----------
+    distance : Distance
+        Distance metric.
+    device: torch.device | str
+        Device to calculate on.
+    batch_size: int
```

Comment:
```suggestion
    batch_size: int = 128
```
Or your opinion on actual number

Reply:
use 128 for now, fixed doc too